content/guides/translation-connection/step3-interact-with-julia.md

```diff
@@ -0,0 +1,241 @@
+:::
+
+Once the translation attribute group and attributes have been created, Julia needs to assign the `Translations` attribute group to all existing `families`.
+
+:::tips
+Julia can do this by using this [bulk action](https://help.akeneo.com/pim/serenity/articles/product-mass-actions.html) process:
+1. Go to `Settings > Families`
+2. Select all families
+3. Click on `BULK ACTIONS button` at bottom
+4. Select `Set attributes requirements` and then click on `NEXT`
+5. Click on `ADD BY GROUPS` drop down option and `ADD` `Translations` attribute group
+6. Click on `CONFIRM`
+:::
+
+>And that's it! Congratulations, everything is now ready to receive Julia's PIM products she wants to translate!
+
+## Julia's translation process
+
+As you have understood, Julia will use the previous PIM attributes to define which of her products are to be translated and how they will be processed.
+
+Now let's see how Julia will work with these new PIM translation features.
+
+### Select the attributes to be translated
+
+First, Julia needs to select product she wants to translate:
+1. Go to `Products` menu
+2. Select the "source" `locale`
+3. Select products (and/or product models) you want to translate
+
+:::tips
+For this product selection, Julia can help herself by using her PIM grid [filtering](https://help.akeneo.com/pim/serenity/articles/products-grid.html#use-filters) system or by creating a [specific view](https://help.akeneo.com/pim/serenity/articles/manage-your-views.html).
+:::
+
+### Perform a "Bulk action"
+
+Then, she needs to perform a [Bulk action](https://help.akeneo.com/pim/serenity/articles/product-mass-actions.html) on these products.
+
+For this action, Julia have to:
+1. Click on `BULK ACTIONS` button at bottom
+2. Select `Edit attribute`s values and click on `NEXT`.
+3. From `SELECT ATTRIBUTES` drop down, search and select these 5 translation attributes and click on `ADD`:
+  * Translation submission name
+  * Translation submitter
+  * Translation queued
+  * Translation locales
+  * Translation due date
+4. Now set the translation attributes:
+  * `Translation name`: enter the desired name for the translation project.
+  * `Translations submitter`: select your name (Julia's name).
+  * `Translation queued`: enable the attribute by sliding the toggle button towards `Yes`.
+  * `Translation locales`: select the desired target locales for translation.
+  * `Translations due date`: Select the date on which you want to receive the translation.
+5. Click on `NEXT` and `CONFIRM` the operation.
+
+After this last action, the `Bulk action` will run and Julia will be notified in the Akeneo PIM notification bar once it is successfully executed.
+
+With the attributes of the products Julia wants to have translated now ready, your connector is ready to go!
+
+### Translation process
```

Comment:
I feel this title is not clear enough. And in fact, this part is quite weird here because it is not part of what Julia has to do. I would go for a dedicated part just after (or even maybe before) the "Julia's translation process" part, named like this "Build your connector" or "Your connector's tasks"...